README.md

```diff
@@ -29,6 +25,104 @@ and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
 If you want to jump in early, you can try building latest code yourself.
 
+## SuperBench capabilities, workflow and benchmarking metrics
+
+The following graphic shows the capabilities provide by SuperBench core framework and its extension.
+
+<img src="imgs/superbench_structure.png">
+
+Benchmarking metrics provided by SuperBench are listed as below.
+
+<table>
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+      </td>
+      <td>
+        <b>Micro Benchmark</b>
+      </td>
+      <td>
+        <b>Model Benchmark</b>
+      </td>
+    </tr>
+    <tr valign="top">
+      <td align="center" valign="middle">
+        <b>Metrics</b>
+      </td>
+      <td>
+        <ul><li><b>Computation Benchmark</b></li>
+          <ul><li><b>Kernel Performance</b></li>
+            <ul>
+              <li>GFLOPS</li>
+              <li>TensorCore</li>
+              <li>cuBLAS</li>
+              <li>cuDNN</li>
+            </ul>
+          </ul>
+          <ul><li><b>Kernel Launch Time</b></li>
+            <ul>
+              <li>Kernel_Launch_Event_Time</li>
+              <li>Kernel_Launch_Wall_Time</li>
+            </ul>
+          </ul>
+          <ul><li><b>Operator Performance</b></li>
+            <ul><li>MatMul</li><li>Sharding_MatMul</li></ul>
+          </ul>
+          <ul><li><b>Memory</b></li>
+            <ul><li>H2D_Mem_BW_&lt;GPU ID&gt;</li>
+              <li>H2D_Mem_BW_&lt;GPU ID&gt;</li></ul>
+          </ul>
+        </ul>
+        <ul><li><b>Communication Benchmark</b></li>
+          <ul><li><b>Device P2P Bandwidth</b></li>
+            <ul><li>P2P_BW_Max</li><li>P2P_BW_Min</li><li>P2P_BW_Avg</li></ul>
+          </ul>
+          <ul><li><b>RDMA</b></li>
+            <ul><li>RDMA_Peak</li><li>RDMA_Avg</li></ul>
+          </ul>
+          <ul><li><b>NCCL</b></li>
+            <ul><li>NCCL_AllReduce</li></ul>
+          </ul>
+        </ul>
+        <ul><li><b>Computation-Communication Benchmark</b></li>
+          <ul><li><b>Mul_During_NCCL</b></li>
+            <ul><li>MatMul_During_NCCL</li><li>RDMA_Avg</li></ul>
+          </ul>
+        </ul>
+        <ul><li><b>Storage Benchmark</b></li>
+          <ul><li><b>Disk</b></li>
+            <ul>
+              <li>Read/Write</li><li>Rand_Read/Rand_Write</li>
+              <li>R/W_Read</li><li>R/W_Write</li><li>Rand_R/W_Read</li><li>Rand_R/W_Write</li>
+            </ul>
+          </ul>
+        </ul>   
+      </td>
+      <td>
+        <ul><li><b>CNN models</b></li>
```

Comment:
@guoshzhao For CNN model, actually we can support all of them. How can we describe here?

Reply:
If we want to cover all benchmarks here, we should list them all according to `BenchmarkRegistry.register_benchmark`:

> https://github.com/microsoft/superbenchmark/blob/e977bbc17f1f9da268498408617d8ecf8ad90ead/superbench/benchmarks/model_benchmarks/pytorch_cnn.py#L147

Reply:
add "other cnn models..." for hot fix